src/couch_replicator_scheduler.erl

```diff
@@ -226,16 +233,21 @@ oldest_job_first(#job{} = A, #job{} = B) ->
 
 not_recently_crashed(#job{} = Job) ->
     case crash_history(Job) of
```

Comment:
if the most recent history item is a `stopped`, we could return true here. That is, we can define that a job that last ran successfully has not recently crashed.